assert/assertions.go

```diff
@@ -262,8 +262,8 @@ func Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{})
 
 	if !ObjectsAreEqual(expected, actual) {
 		diff := diff(expected, actual)
-		return Fail(t, fmt.Sprintf("Not equal: %#v (expected)\n"+
-			"        != %#v (actual)%s", expected, actual, diff), msgAndArgs...)
+		return Fail(t, fmt.Sprintf("Not equal: %#v (%[1]T (expected)\n"+
```

Comment:
Fixed in 924cf70. I actually wanted the type info in parentheses, so I added the missing ones